### ISSUE TITLE
Support for Swedish language

### ISSUE THREAD

**Issue by EmilStenstrom** [kermitt2/entity-fishing#149]
Hi!

In the list of languages I don't see Swedish. It's a small language, but has a very big wikipedia with ~2.5M articles. Can entity-fishing be trained on swedish, or is there some deeper reason that it's not included?

**Comment by kermitt2** [comment]
Hi @EmilStenstrom !

Thank you for the request. Swedish should work well indeed given the size of its Wikipedia. I think it's the largest one not support by entity-fishing yet, with Dutch. It will try to include it in the next batch of supported languages.

**Comment by EmilStenstrom** [comment]
That sounds awesome! Looking forward to testing it! :)

**Comment by kermitt2** [comment]
![Screenshot from 2023-01-20 18-44-34](https://user-images.githubusercontent.com/2340795/213770232-6842b89d-1703-48db-95d4-d9c817ecf90a.png)

**Comment by EmilStenstrom** [comment]
Nice! Happy to see it disambiguate Swedish. Looking at that specific example, the things it mentions are not entities, but they are “concepts”. Translated: “year”, “consumption”, “health”. Is that intentional?

**Comment by kermitt2** [comment]
Yes that's the goal, every Wikidata entities is disambiguated, based on the Wikipedia anchors - Wikidata calls "entities" the concepts and their instances. We can then refine based on the statements P279 and P31 to select what's wanted for a given task/application. Another one more:

![Screenshot from 2023-01-21 21-16-59](https://user-images.githubusercontent.com/2340795/213885667-a920853d-f228-42c3-b90d-d7b238f27b3e.png)

**Comment by EmilStenstrom** [comment]
Awesome! Using wikidata statements to select what you want is super powerful. Eager to try this out when 0.0.6 is released! :)
